app/org/iatoki/judgels/raguel/services/impls/ThreadPostServiceImpl.java

```diff
@@ -164,6 +199,19 @@ public void createPost(ForumThread forumThread, String userJid, String subject,
 
         postContentDao.persist(postContentModel, userJid, userIpAddress);
 
+        UserPostCountModel userPostCountModel;
+        try {
```

Comment:
Please write a method `existsByUserJid()` in the DAO instead of using try-catch. This is the idiomatic way in Judgels.